edge-agent/test/Microsoft.Azure.Devices.Edge.Agent.Kubernetes.Test/edgedeployment/pvc/KubernetesPvcByValueComparerTest.cs

```diff
@@ -139,79 +140,94 @@ public void FieldComparisonTests()
         [Fact]
         public void SuccessComparisonTest()
         {
-            var x = new V1PersistentVolumeClaim
+            (var x, var y) = MakeIdenticalPVCs();
+            Assert.True(Comparer.Equals(x, y));
+        }
+
+        [Fact]
+        public void DistictionTest()
+        {
+            (var x, var y) = MakeIdenticalPVCs();
+            var distinct = new List<V1PersistentVolumeClaim> { x, y }.Distinct(Comparer);
```

Comment:
```suggestion
            var claims = new List<V1PersistentVolumeClaim> { x, y };
            
            var distinct = claims.Distinct(Comparer);
```

Reply:
Interesting. VS suggested `new List<V1PersistentVolumeClaim> { x, y }.Distinct(Comparer);` 
I write it as 2 lines.

Reply:
Yeah, it's kinda Arrage, Act, Assert thing